### PR TITLE
[HNC-475] - Citadel Scan Added

### PR DESCRIPTION
**Pull request by vganpisetty**
Adding Citadel step

### CODE REVIEW

**Review by cardosov — CHANGES_REQUESTED** [review]
Why is the Citadel config file not being used? Isn't this suppose to be the same as used in the [samples](https://github.com/lumada-common-services/honeycomb-samples/blob/master/.github/workflows/gradle_merge.yaml#L67-L74)? Is there a PR for `pdi-plugins` to include this config file?

File: .github/workflows/merge.yml [hunk 1]
```diff
@@ -240,6 +241,15 @@ jobs:
       #    BlackDuck_Api_Token: "${{ secrets.ORION_BLACKDUCK_TOKEN }}"
       #    BlackDuck_Url: "${{ inputs.blackduck_server_url }}"
```
Comment (cardosov) [inline .github/workflows/merge.yml:242]: If adding the citadel step to this, then we can remove the blackduck one.

File: .github/workflows/merge.yml [hunk 2]
```diff
@@ -240,6 +241,15 @@ jobs:
...
       #    ADDITIONAL_ARGS: ${{ inputs.blackduck_additional_args }}
+    
+      - name: Trigger Citadel scan
+        id: Citadel
+        uses: lumada-common-services/gh-composite-actions@stable
+        env:
+          TRIGGER_CITADEL_SCAN: true
+          CITADEL_URL: 'https://citadel.wal.eng.hitachivantara.com/citadel' 
+          SECRETS_CONTEXT: ${{ toJson(secrets) }}
```
Comment (cardosov) [inline .github/workflows/merge.yml:251]: Is there a build run that shows this working?

**Review by cardosov — CHANGES_REQUESTED** [review]

File: .github/workflows/merge.yml (hunk 1 above)
Comment (vganpisetty) [reply .github/workflows/merge.yml:242]: we just kept Blackduck commented just for future use

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Pull request closed without merging**